### ISSUE TITLE
if ned model.eval() when export model?

### ISSUE THREAD

**Issue by jeannotes** [open-mmlab/mmsegmentation#1464]
after sef model.eval() after build model from config, when export to libtorch using your pytorch2libtorch, if I reload model from the jid script model, I just find that the result is not right?
if I build model from config and just load checkpoint, not set model.eval(), the result is right,
and why is that?

**Comment by RunningLeon** [comment]
@jeannotes Hi, normaly we set to eval mode when converting pytorch model to other format. Could you provide what script you run and what the difference is?
